apps/toolbox.fr.i18n

```diff
@@ -514,3 +514,17 @@ HartreeConstantTag = "Constante de Hartree"
 MagneticFluxQuantumTag = "Quantum de Flux Magnétique"
 ConductanceQuantumTag = "Quantum de Conductance"
 CirculationQuantumTag = "Quantum de Circulation"
+Logic = "Logic"
+LogicalBitClear = "Effacer le bit numéro b dans a"
+LogicalBitFlip = "Inverser le bit numéro b dans a"
+LogicalBitGet = "Obtenir le bit numéro b dans a"
+LogicalBitSet = "Allumer le bit numéro b dans a"
+LogicalBitsClear = "Effacer bits b ds a [a AND NOT b]"
+LogicalShiftLeft = "Décalage logique gauche [a << s]"
+LogicalShiftRightArithmetic = "Décalage arith. droite [a >>> s]"
+LogicalShiftRight = "Décalage logique droite [a >> s]"
+LogicalRotateLeft = "Rotation gauche de a par r bits"
+LogicalRotateRight= "Rotation droite de a par r bits"
+TwosComplementToBits = "Equivalent en complément à deux"
+CeilingLog2 = "Nombre de bits nécessaires pour stocker a"
```

Comment:
The text is too long (maybe shortening Nombre to nb would help)

Reply:
'Nombre de bits nécessaires pour stocker a' is now 'Nb de bits nécessaires pour stocker a'